sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/v2/parquet/ParquetPartitionReaderFactory.scala

```diff
@@ -72,6 +73,8 @@ case class ParquetPartitionReaderFactory(
   private val enableOffHeapColumnVector = sqlConf.offHeapColumnVectorEnabled
   private val enableVectorizedReader: Boolean =
     ParquetUtils.isBatchReadSupportedForSchema(sqlConf, resultSchema)
+  private val supportsColumnar = enableVectorizedReader && sqlConf.wholeStageEnabled &&
```

Comment:
https://github.com/apache/spark/blob/222cb80227fb9c24b1677c5a746c0a174a18b537/sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/parquet/ParquetFileFormat.scala#L168-L173

Should we also remove `conf.parquetVectorizedReaderEnabled` from `ParquetFileFormat#supportBatch`? `ParquetUtils.isBatchReadSupportedForSchema` contains this condition

Reply:
Yeah good catch, I can remove that